src/main/resources/templates/search.html

```diff
@@ -38,6 +38,34 @@ <h2>Search by:</h2>
 
     <!-- TODO #4 - Loop over the search results to display all job fields. -->
 
+    <table class="job-listing" th:each="job: ${jobs}">
+        <tr>
+            <td>ID</td>
+            <td th:text="${job.getId()}"></td>
+        </tr>
+        <tr>
+            <td>Name</td>
+            <td th:text="${job.getName()}"></td>
+        </tr>
+        <tr>
+            <td>Employer</td>
```

Comment:
On your `master` branch, it seems that the closing tag was missing its left angle bracket. I don't know if your code was updated since I pulled it today, but it looked like this and caused a test to TestTaskTwo to fail: `<td>Employer/td>`.